### ISSUE TITLE
is this project still active?

### ISSUE THREAD

**Issue by mewalig** [mijia/modelq#25]

**Comment by kanocz** [comment]
@mewalig what do you mean?.. pull requests are merged very fast, no major issues exists...

**Comment by mijia** [comment]
@mewalig Hi, I'm currently working on other projects, but I can still handle the pull requests or issues in the spare time.

Any ideas or PRs are appericated.

**Comment by mewalig** [comment]
Thanks. Are there any plans to add support for sqlite3 and if not would you consider it?

**Comment by mijia** [comment]
@mewalig Yeah, I have thought about sqlite3 back then. For now, 'modelq' depends on the scheme definition or metadata from the db, such as we fetch scheme information from mysql or postgres. I knew that you can get sqlite3 scheme from the CLI by `.scheme` but I didn't find way to get that data through api calls.

If you are familiar with that, maybe you can help me to give me some approach to poke, :)

**Comment by mewalig** [comment]
You can query metadata with a standard select call e.g. "SELECT * FROM dbname.sqlite_master". I think it's a subset of what you already have and so would be very straightforward to support-- you'd only have to change the calls you use to query the data dictionary and remove any functionality that doesn't apply to sqlite
